tests/integration/targets/hcloud_iso_info/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+collections:
+  - community.general.ipfilter
```

Comment:
What is the ipfilter used for?

Reply:
This is a good question, I copied this from another test case.
- The collection name seems invalid anyway,
- The name/function is not used anywhere.

I'll try to find out why we have this.